_posts/2020-09-02-cockpit-227.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Cockpit 227
+author: pitti
+date: 2020-09-02
+tags: cockpit linux
+slug: cockpit-227
+category: release
+summary: Cockpit with VM filtering and PatternFly 4 design changes
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 227.
+
+### Machines: VM list filtering
```

Comment:
```suggestion
### Machines: Virtual machine list filtering
```

As the headings are used in the shortened release notes, we should make them stand alone. It's best to expand acronyms in that case.